main-client.js

```diff
@@ -80,8 +80,12 @@ export const createMeteorNetworkInterface = (givenConfig) => {
   return networkInterface;
 };
 
-export const meteorClientConfig = (networkInterfaceConfig) => {
+export const meteorClientConfig = (globalConfig = {}) => {
+  // destructure general apollo client configuration from network interface configuration
+  const { queryDeduplication = false, ...networkInterfaceConfig } = globalConfig;
```

Comment:
I got scared at first! 😱 It's has intended not `undefined` but maybe it should be done some other way to be clearer ?

Something approximatively like: 
```js
export const meteorClientConfig = ({
  networkInterface, // expected config object by Apollo Client
  ...clientConfig,  
} = {
  networkInterface: createMeteorNetworkInterface()
}) => {

...

}
```